Makefile

```diff
@@ -104,15 +104,11 @@ help:  ## Display this help
 .PHONY: unit
 unit: $(SETUP_ENVTEST) ## Run unit test
 	$(shell $(SETUP_ENVTEST) use -p env --os $(ENVTEST_OS) --arch $(ARCH) $(ENVTEST_K8S_VERSION)); \
```

Comment:
You need to change all of the `; \` to `&& \` to chain them only when previous one succeeds. This allows make to catch error in earlier commands, otherwise it will only fail if the last command fails.

Similar '; \` chains are also present in other targets as well, please fix them as well.